### ISSUE TITLE
install.package("wk") not working - non-ASCII characters in lib-path

### ISSUE THREAD

**Issue by AndreM84** [paleolimbot/wk#93]
Dear @paleolimbot,

unfortunatly, I cannot manage to install the wk-package.

```
install.packages("wk")
Installing package into ‘C:/Users/André/Documents/R/win-library/3.6’
(as ‘lib’ is unspecified)

  There is a binary version available but the source version is later:
   binary source needs_compilation
wk  0.4.1  0.5.0              TRUE

installing the source package ‘wk’

versuche URL 'https://cran.rstudio.com/src/contrib/wk_0.5.0.tar.gz'
Content type 'application/x-gzip' length 138686 bytes (135 KB)
downloaded 135 KB

* installing *source* package 'wk' ...
** Paket 'wk' erfolgreich entpackt und MD5 Summen überprüft
** using staged installation
Warnung in file(file, if (append) "a" else "w")
  kann Datei 'C:/Users/Andri/Documents/R/win-library/3.6/00LOCK-wk/00new/wk/DESCRIPTION' nicht Ã¶ffnen: No such file or directory
Error in file(file, if (append) "a" else "w") : 
  kann Verbindung nicht Ã¶ffnen
ERROR: installing package DESCRIPTION failed for package 'wk'
* removing 'C:/Users/André/Documents/R/win-library/3.6/wk'
Warning in install.packages :
  installation of package ‘wk’ had non-zero exit status
```

As you can see, the lib-path is not shown correctly, when the description file shall be read. The non-ASCII "é" became an "i" . The installation with 
`install.packages("wk", lib = "C:/Users/André/Documents/R/win-library/3.6")`
throws the same error.

Here same additional information on my system from sessionInfo()

```
> sessionInfo()
R version 3.6.2 (2019-12-12)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)
```

Is there any work around available for manually processing the installation routine and checking and correcting the lib-path each time a "wrong" encoding happend?

Thank in advance and best regards
André

**Comment by AndreM84** [comment]
Update: 
I started a workaround according to this Stackoverflow answer
https://stackoverflow.com/a/19662905/12925302

- I set a new User Environment Variable for R_Lib_User wiht a path without a special character ("C:/R/win-library/3.6").
- Now, I have to install tons of packages, which I used in the past. Fortunately, most of my scripts do check whether packages are installed and - if not - install them quitely in the background before loading them with library().

For whatever reason, RStudio shows all packaes as SystemLibrary now, while before there was a seperation bewteen the User Library and the System Library. (The packages are stored in two seperate folders)
`> .libPaths()
[1] "C:/R/win-library/3.6"               "C:/Program Files/R/R-3.6.2/library"`
I hope this does not have an negative impact in the furture.

Maybe a fix of the special character issue would help others to avoid this.
